demos/time_series_forecasting_demo/python/time_series_forecasting_demo.py

```diff
@@ -31,26 +31,23 @@ class ForecastingEngine:
 
     Arguments:
         model_xml (str): path to model's .xml file.
-        model_bin (str): path to model's .bin file.
         input_name (str): name of input blob of model.
         output_name (str): name of output blob of model.
     """
-    def __init__(self, model_xml, model_bin, input_name, output_name, quantiles):
+    def __init__(self, model_xml, input_name, output_name, quantiles):
         device = "CPU"
         log.info('OpenVINO Inference Engine')
         log.info('\tbuild: {}'.format(get_version()))
-        self.ie = IECore()
+        core = Core()
         log.info('Reading model {}'.format(model_xml))
-        self.net = self.ie.read_network(
-            model=model_xml,
-            weights=model_bin
-        )
-        self.net_exec = self.ie.load_network(self.net, device)
+        self.model = core.read_model(model_xml)
+        compiled_model = core.compile_model(self.model, device)
+        self.infer_request = compiled_model.create_infer_request()
         log.info('The model {} is loaded to {}'.format(model_xml, device))
-        self.input_name = input_name
-        self.output_name = output_name
+        self.input_tensor_name = input_name
+        self.output_tensor_name = output_name
         self.quantiles = quantiles
-        assert self.output_name != "", "there is not output in model"
+        assert self.output_tensor_name != "", "there is not output in model"
```

Comment:
strange check, especially if consturctor required to provide name :)

Reply:
Removed the assert